src/NuGet.Core/NuGet.Commands/SourcesCommands/SourceRunners.cs

```diff
@@ -336,6 +336,27 @@ public static ISettings GetSettings(string configfile)
             }
         }
 
+        public static ISettings GetSettingsFromDirectory(string directory)
+        {
+            if (Directory.Exists(directory))
+            {
+                return NuGet.Configuration.Settings.LoadDefaultSettings(directory,
+                   configFileName: null,
+                   machineWideSettings: new XPlatMachineWideSetting());
```

Comment:
```suggestion
                return NuGet.Configuration.Settings.LoadDefaultSettings(
                   directory,
                   configFileName: null,
                   machineWideSettings: new XPlatMachineWideSetting());
```

Nit: If one arg is on a new line, all args should be.